lib/navy/renderer.rb

```diff
@@ -94,7 +94,12 @@ def render_section(section, level, parent_id)
 
     
     def div_tag(options, &block)
-      content_tag_string(:div, block.call, options, true)
+      if ActionPack.version.to_s < '5'
```

Comment:
I think `ActionPack.version` does not exist in old Railses. I believe you can use `ActionPack::VERSION::MAJOR`.

Reply:
Thank you, I was able to verify that your proposed variant is backwards compatible to at least ActionPack 2. I did not try it on v1.

Reply:
Did not try AP 1, but it seems to fail on AP 3.

Reply:
I think we mean the same thing:
`ActionPack::VERSION::MAJOR` is compatible with Rails 2+
`ActionPack.version` is compatible with Rails 4+
:slightly_frowning_face: